cpp/src/arrow/adapters/orc/CMakeLists.txt

```diff
@@ -20,7 +20,8 @@
 #
 
 # Headers: top level
-install(FILES adapter.h DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/arrow/adapters/orc")
+install(FILES adapter.h adapter_options.h adapter_common.h
```

Comment:
Hmm, I know that we started this way, but I don't think it makes sense to have "adapter" in the filename, especially as it's already in the directory path. Can you name those `options.h` and `common.h`, simply?

Reply:
Thanks! Done!